src/renderer.rs

```diff
@@ -14,7 +14,7 @@ use tokio::io::AsyncWriteExt;
 pub trait GraphvizRenderer {
     async fn render_graphviz<'a>(
         block: GraphvizBlock,
-        _config: &GraphvizConfig,
+        config: &GraphvizConfig,
```

Comment:
Thanks. Left that in in a previous change by accident and hadn't gotten around to fixing it.